Source/WebCore/style/StyleFontSizeFunctions.h

```diff
@@ -39,6 +40,8 @@ enum class MinimumFontSizeRule : uint8_t { None, Absolute, AbsoluteAndRelative }
 float computedFontSizeFromSpecifiedSize(float specifiedSize, bool isAbsoluteSize, float zoomFactor, MinimumFontSizeRule, const Settings::Values&);
 float computedFontSizeFromSpecifiedSize(float specifiedSize, bool isAbsoluteSize, bool useSVGZoomRules, const RenderStyle*, const Document&);
 float computedFontSizeFromSpecifiedSizeForSVGInlineText(float specifiedSize, bool isAbsoluteSize, float zoomFactor, const Document&);
+float adjustedFontSize(float size, float sizeAdjust, float xHeight);
```

Comment:
I suggest making this overload of `adjustedFontSize` be private and not declared in the header.